### PR TITLE
Refactor coloring database/collection names

### PR DESCRIPTION
**Pull request by pvdb**
Hey @TylerBrock, this PR contains some refactoring _(but no functional changes)_ to the way database and collection names are colorized for the various `show` and `count` shell commands... as a consequence of this refactoring, the `printPaddedColumns()` function is simplified, as it no longer needs to concern itself with coloring any of its columns... this refactoring is prep-work for enabling the `printPaddedColumns()` function to print out multiple columns _(instead of just printing out two, as it currently does)_.

Basically it's a bit of refactoring to enable the multi-column output/formatting we need for #148

### CODE REVIEW

**Comment by TylerBrock** [comment]
Ok, this seems great. Thanks.

**Comment by pvdb** [comment]
Awesome, next stage will the "multi-column output"!

**Comment by pvdb** [comment]
> Awesome, next stage will be the "multi-column output"!

That's done in #150, ready for review and sign off!  :smile:
